todo/management/commands/hopper.py

```diff
@@ -136,7 +146,16 @@ def add_details(self, build, extracted, **kwargs):
         fake = Faker()  # Use to create user's names
         taskgroup = self.task_list.group
 
-        self.created_by = taskgroup.user_set.all().order_by("?").first()
+        # Django's ManyToManyRel is a little odd in that its directional sense 
+        # is not guranteeed and must be tested for. One of these models is User, 
+        # the other is the Groups model (the one group_field points to). We seek 
+        # the attribute with which we can access the set of users in a group.
+        user_model = get_user_model()
+        group_field = getattr(user_model, setting("TODO_USER_GROUP_ATTRIBUTE"), "groups")
```

Comment:
Not DRY

Reply:
Again, can go to a utility function. Will do that on all these repeats. Agreed. DRY is better.